src/options.cpp

```diff
@@ -1541,7 +1541,7 @@ void options_manager::add_options_general()
     add_empty_line();
 
     add( "SAFEMODE", "general", to_translation( "Safe mode" ),
-         to_translation( "If true, will hold the game and display a warning if a hostile monster/NPC is approaching." ),
+         to_translation( "If true, will hold the game and display a warning if a hostile monster/NPC is approaching.  This will also prevent attacking neutral creatures on move." ),
```

Comment:
imo don't put it here, players should not be normally messing with options for attacking neutral creatures and the message they're given already guides them to correct way to do it, you could show them their keybind like done in https://github.com/CleverRaven/Cataclysm-DDA/blob/3e920a092b4136c9ecec0b9f0ab3f90a4ced327f/src/game.cpp#L9930